src/components/movie-view/movie-view.jsx

```diff
@@ -7,65 +7,63 @@ import { MovieCard } from "../movie-card/movie-card";
 
 export const MovieView = ({ movies, user, token, updateUser }) => {
   const { movieId } = useParams();
-  const movie = movies.find(m => m.id === movieId);
+  const movie = movies.find((m) => m.id === movieId);
+  console.log(JSON.stringify(user, null, 2));
```

Comment:
@Jaime-Anderson  Delete this line.

Readmore here
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify